independent-publisher-connectors/HubSpot CRM/apiDefinition.swagger.json

```diff
@@ -0,0 +1,4553 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Default title",

```

Comment:
@troystaylor Please provide a valid title

Reply:
The title I see is:
"HubSpot CRM is a system for managing customer relationships and storing data. The CRM custom connector provides access to CRM objects or groups of objects, which are represented as a map of property names to values."

Reply:
> The title I see is:
> "HubSpot CRM is a system for managing customer relationships and storing data. The CRM custom connector provides access to CRM objects or groups of objects, which are represented as a map of property names to values."

This is the description value.

The title value need to be changed.

line number 4:  `"title": "Default title"`